test/Serilog.ApprovalTests/Serilog.approved.txt

```diff
@@ -830,7 +832,7 @@ namespace Serilog.Parsing
     }
     public sealed class PropertyToken : Serilog.Parsing.MessageTemplateToken
     {
-        public PropertyToken(string propertyName, string rawText, string? format = null, in Serilog.Parsing.Alignment? alignment = null, Serilog.Parsing.Destructuring destructuring = 0) { }
```

Comment:
Ditto here.

Reply:
`Alignment` is a readonly struct and works well with `in`m but `Nullable<Alignment` **is not** a readonly struct so `in` creates defensive copy as I understand.

> The in keyword was added here thanks to advice from a competing analyzer, IIRC 

No, I added in myself when making struct readonly.

Reply:
Adding or removing `in` from a parameter is a breaking change according to https://learn.microsoft.com/en-us/dotnet/core/compatibility/library-change-rules, I'd like to remove this but might need to wait for v5.